### ISSUE TITLE
Silent reply for gifs

### ISSUE THREAD

**Issue by adyuwu** [zeGolem/nyabot#6]
The bot currently pings the user with a reply to the gif command.
It may be more ideal for it to have a _silent_ reply, without pinging the person sending the command.
Attached is a screenshot of what it currently looks like. Also, the visual difference between a non-ping reply and a ping reply.
<img width="441" alt="Screenshot 2023-03-18 at 23 45 16" src="https://user-images.githubusercontent.com/123894283/226145819-a81aaad0-4e1a-46cc-b7f7-55b4976bf148.png">
<img width="372" alt="Screenshot 2023-03-18 at 23 47 16" src="https://user-images.githubusercontent.com/123894283/226145821-efcb175d-9c89-4add-9dd3-de7efab47672.png">

**Comment by zeGolem** [comment]
good idea, didn't think of that :) will see if i can implement that once #5 is merged!
